### PR TITLE
Phase2-hcx214 Add a new scenario and workflow with latest Phase2 configuration including HFNose

### PR DESCRIPTION
**Pull request by bsunanda**
#### PR description:

Add a new scenario (2026D47) and its corresponding workflow 22434 with latest version of Phase2 geometry together with HFNose

#### PR validation:

Tested the workflow 22434.0

#### if this PR is a backport please specify the original PR:

No back porting is needed

### CODE REVIEW

**Comment by kpedro88** [comment]
please test workflow 22434.0

**Comment by kpedro88** [comment]
@bsunanda be aware this PR will have to be updated after #27632 is merged

**Comment by cvuosalo** [comment]
+1

**Comment by kpedro88** [comment]
@bsunanda please update this to the latest IB and rerun the geometry script to account for the change in  #27632

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by ianna** [comment]
+1

File: Configuration/PyReleaseValidation/python/relval_2026.py [hunk 1]
```diff
@@ -23,5 +23,6 @@
 numWFIB.extend([21234.0]) #2026D44
 numWFIB.extend([21634.0]) #2026D45
 numWFIB.extend([22034.0]) #2026D46
+numWFIB.extend([22434.0]) #2026D47
```
Comment (fabiocos) [inline Configuration/PyReleaseValidation/python/relval_2026.py]: @bsunanda @kpedro88 this inflates further the list of geometry scenarios that we would regularly run in tests. Do we need at the same time D45, D46 and D47? Do you have a plan for the scenarios evolution and final target?

**Comment by ianna** [comment]
@kpedro88 - please review

**Comment by ghugo83** [comment]
Haha I was about to have a D47 as well ;p
I will wait until this PR is discussed / reviewed, then create a PR with skewed Inner Tracker and associated geometry scenario / workflows.

**Comment by fabiocos** [comment]
@ghugo83 could you please clarify the updates you need? We should try to avoid multiplying scenarios, I would like we agree with @kpedro88 as well a strategy about next future needs.

**Comment by ghugo83** [comment]
@fabiocos @kpedro88 

I have the GDML description of the skewed Inner Tracker ready. Let's call this Tracker T16. Several ladders are rotated in TBPX around CMX_Z axis ('skewed'), to allow space for cooling pipes, as it will be in the detector to be installed.
There is no update in terms of Material Budget between T15 and T16. The diff is in the geometry.

T16 is the Phase 2 Tracker closest to production at the present time. Hence, it would be best to have it integrated in official CMSSW, and no private branch.
A possibility is to introduce a D48 geometry scenario with T16 (or D47, depending on what is decided here in this PR).

I understand the need to avoid having too many geometry scenarios though. Am open to any idea you suggest, as long as it allows T16 to be integrated in official CMSSW, and compared with T15 :)

NB: - It was decided to keep T14 in CMSSW for some time. Indeed, the eventual Tracker MB should be in between T14 and T15-T16.
- Once T16 is debugged / potentially corrected / validated, T15 could in theory be dropped (but obviously might still be in used in several studies).

@emiglior @mmusich

File: Configuration/PyReleaseValidation/python/relval_2026.py (hunk 1 above)
Comment (kpedro88) [reply Configuration/PyReleaseValidation/python/relval_2026.py]: It is not included in the PR tests, only the IB tests. To me, this is reasonable. We will try to keep the PR tests to a minimum, but the IB should test every workflow (in my opinion).

**Comment by kpedro88** [comment]
+upgrade

**Comment by kpedro88** [comment]
@ghugo83 I need to think a bit about the best approach for the tracker going forward. Can you open a separate issue for us to discuss this?

**Comment by bsunanda** [comment]
@prebello @pgunnell @franzoni @zhenhu Could you please approve this PR

**Comment by ghugo83** [comment]
@kpedro88 Ok as discussed with you privately, am going to create a PR with a D48 scenario now.
Can open a discussion for further Phase 2 Tracker updates strategy.

The Phase 2 Tracker R&D is more mature though now anyway. Of course, still up to installation and even after, Phase 2 Tracker updates are to be expected though, but hopefully at a lower frequency.

File: Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py [hunk 2]
```diff
@@ -368,6 +370,13 @@
         'Era' : 'Phase2C9_timing_layer_bar',
         'ScenToRun' : ['GenSimHLBeamSpotFull','DigiFullTrigger','RecoFullGlobal', 'HARVESTFullGlobal'],
     },
+    '2026D47' : {
+        'Geom' : 'Extended2026D47',
+        'HLTmenu': '@fake2',
+        'GT' : 'auto:phase2_realistic',
```
Comment (ghugo83) [inline Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py]: Hmmm there is an issue here, no?

You need to add: 
'auto:phase2_realistic_T15'

See https://github.com/cms-sw/cmssw/pull/27517/ from @mmusich

Comment (mmusich) [reply Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py]: yes this needs to be changed, but as it was not clear this PR was going to be merged at all I didn't comment.

Comment (bsunanda) [reply Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py]: Corrected that part now

**Comment by kpedro88** [comment]
please test workflow 22434.0

**Comment by kpedro88** [comment]
+upgrade

**Comment by chayanit** [comment]
+1

**Comment by cvuosalo** [comment]
+1

**Comment by kpedro88** [comment]
+1

**Pull request merged**